compliance/http/src/test/java/org/eclipse/rdf4j/http/server/ProtocolTest.java

```diff
@@ -482,6 +516,62 @@ public void run() {
 		threadPool.shutdownNow();
 	}
 
+	private Set<Namespace> getNamespaces(String location)
+		throws Exception
+	{
+		URL url = new URL(location);
+		HttpURLConnection conn = (HttpURLConnection)url.openConnection();
+		try {
+			conn.setRequestProperty("Accept", "text/csv; charset=utf-8");
+			conn.connect();
+
+			InputStream connIs = conn.getInputStream();
+			try {
+				int responseCode = conn.getResponseCode();
+				if (responseCode != HttpURLConnection.HTTP_OK) {
+					String response = "location " + location + " responded: " + conn.getResponseMessage()
+							+ " (" + responseCode + ")";
+					fail(response);
+				}
+
+				String responseText = IOUtil.readString(connIs);
+				String[] rows = responseText.split("\\r?\\n");
```

Comment:
Hi @ansell 

I decided to use CSV because it is relatively easily to parse, but in case I can use an existing parser I could opt to use another format, maybe the XML serialization, in order to avoid the introduction of another dependency.

I think that I could work on this modification tomorrow or the day after tomorrow.